### DISCUSSION TITLE
Ollama

### DISCUSSION THREAD

**Discussion by kusold** [selfhst/icons#155] [New Icon Request]
Ollama allows you to run LLMs locally, with API Access

SVG: https://github.com/ollama/ollama/issues/2152#issuecomment-1905286922

PNG (from the [readme](https://github.com/ollama/ollama/blob/aed1419c64748f31e66fe04875a6696b70761038/README.md?plain=1#L2)): https://github.com/ollama/ollama/assets/3325447/0d0b44e2-8f4a-4e99-9b52-a5c1c741c8f7

**Comment by shollyethan** [comment]
Thanks for the SVG link - I was able to recreate the favicon from the Ollama website with it, which can now be found in the collection!
